src/main/scala/com/fulcrumgenomics/bam/FilterBam.scala

```diff
@@ -51,6 +52,11 @@ import htsjdk.samtools.util.IntervalList
      |marked as secondary alignments, reads marked as duplicates, and if a set of Intervals are provided,
      |reads that do not overlap any of the intervals.
      |
+     |The `--remove-unpaired-reads`, which removes reads that are not part of a mapped pair will remove
```

Comment:
I'm struggling to come up with a good name for this that's not really long and is in the style of other options.  I like `--remove-*` because I think it's clearer that the option removes those things, rather than `--keep-*` because that implies things will be kept, which other options may invalidate.

`--remove-reads-not-part-of-mapped-pairs` seems overly wordy, but clearest.  Perhaps just expanding the documentation for that option is best?